okapi-common/src/main/java/org/folio/okapi/common/logging/FolioLoggingContext.java

```diff
@@ -0,0 +1,70 @@
+package org.folio.okapi.common.logging;
+
+import io.vertx.core.Context;
+import io.vertx.core.Vertx;
+import org.apache.logging.log4j.core.LogEvent;
+import org.apache.logging.log4j.core.config.plugins.Plugin;
+import org.apache.logging.log4j.core.lookup.StrLookup;
+
+/**
+ * This class should be used for storing context variables
+ * and use them in logging events.
```

Comment:
Can you be more specific about the usage of this class? It should mention log4j, Vertx context, Vertx vertx local data.